assets/js/voronoi.js

```diff
@@ -26,6 +32,11 @@ const dropArea = document.getElementById("dropzone");
 dropArea.addEventListener("drop", handleDrop, false);
 dropArea.addEventListener("input", handleChange, false);
 
+numPointsSlider.oninput = function () {
+  pointsToUse = 10000 - this.value || 1;
```

Comment:
Better use `event.target.value` it's more safe in cave of losing context.
I don't understand what do you want to achieve with this `|| 1` in the end. Do you expect `value` to be undefined?

Reply:
if "e.target.value" is ever zero, the algorithm breaks because there can't be zero points. This `|| 1` ensures it will always at least still return 1 point to run the algo with.